main/com.spotify/eval/scenarios.txt

```diff
@@ -344,7 +344,7 @@ A: >> expecting = null
 U: \t $dialogue @org.thingpedia.dialogue.transaction.execute;
 U: now => @com.spotify.create_playlist(name="new playlist");
 
-A: I created a new playlist for you named new playlist\.
+A: I created a new playlist named new playlist\.
```

Comment:
make that an optional clause instead of removing the clause, in case the agent decides to include it again in the future (it's a bit non-deterministic at the moment)